helm/idol-library/templates/aciserver/_container.tpl

```diff
@@ -18,15 +18,21 @@
 {{- $env := get . "env" | default list -}}
 {{- $volumeMounts := get . "volumeMounts" | default list -}}
 {{- $ports := get . "ports" | default list -}}
+{{- $useDefaultProbe := dig "useDefaultProbe" true . -}}
+{{- $setACIPorts := dig "setACIPorts" true . -}}
```

Comment:
Would prefer if we had one guard variable here detected from presence or otherwise of `$component.aciPort`